lib/ClangImporter/ImportType.cpp

```diff
@@ -688,6 +693,10 @@ namespace {
           break;
         }
 
+        static llvm::StringLiteral vaListNames[] = {
```

Comment:
Oops, I meant to make this `const`. It'll probably be optimized as such anyway, but still.